diff -up step-25.08.3/stepcore/types.h.omv~ step-25.08.3/stepcore/types.h
--- step-25.08.3/stepcore/types.h.omv~	2025-11-06 20:06:28.250533619 +0100
+++ step-25.08.3/stepcore/types.h	2025-11-06 20:07:49.187740389 +0100
@@ -38,7 +38,7 @@ struct Color
 
 template<> inline QString typeToString(const Color& v)
 {
-    return QStringLiteral("#%1").arg(v, 8, 16, QLatin1Char('0'));
+    return QStringLiteral("#%1").arg(static_cast<unsigned int>(v), 8, 16, QLatin1Char('0'));
 }
 
 template<> inline Color stringToType(const QString& s, bool *ok)
